core-java-modules/core-java-8-2/src/main/java/com/baeldung/interfaceVsAbstractClass/CircleInterface.java

```diff
@@ -0,0 +1,18 @@
+package com.baeldung.interfaceVsAbstractClass;
+
+import java.util.Arrays;
+import java.util.List;
+
+public interface CircleInterface {
+    List<String> allowedColors = Arrays.asList("RED", "GREEN", "BLUE");
+
+    String getColor();
+    
+    public default boolean isValid() {
+        if (allowedColors.contains(getColor())) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
You could simplify this to a one-liner: `return allowedColors.contains(getColor());`